src/utils.rs

```diff
@@ -10,3 +11,70 @@ pub fn read_file<P: AsRef<Path>>(file_name: P) -> Result<String> {
     file.read_to_string(&mut data)?;
     Ok(data)
 }
+
+/// Execute a command and return the output on stdout and stderr if sucessful
+pub fn exec_cmd(cmd: &str, args: &[&str]) -> Option<(String, String)> {
```

Comment:
I think that working with a tuple here may make understanding how it's used a little more difficult than having a struct with `stdout` and `stderr`.

In most cases it would be used like this:
```rust
const version = exec_cmd("npm", &["--version"])?.0;
```

Reply:
Have switched this over to return a simple struct with fields `stdout` and `stderr`, usage would now be something like
```rust
let ruby_version = utils::exec_cmd("ruby", &["-v"])?.stdout;
```